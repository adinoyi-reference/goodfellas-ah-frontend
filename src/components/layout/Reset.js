import React from 'react';
import AuthImage from '../Auth/AuthImage';

const Auth = (props) => {
  const { children } = props;
  return (
    <div className="auth-flex">
      <AuthImage />
      <div className="auth-form">
        {children}
      </div>
    </div>
  );
};

export default Auth;
