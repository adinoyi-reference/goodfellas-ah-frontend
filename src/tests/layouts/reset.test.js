import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { MemoryRouter } from 'react-router-dom';
import Reset from '../../components/layout/Reset';
import Root from '../../root';

let wrapped;

beforeEach(() => {
  wrapped = mount(

    <Root>
      <MemoryRouter initialEntries={[{ key: 'testKey' }]}>
        <Reset />
      </MemoryRouter>
    </Root>,

  );
});
afterEach(() => wrapped.unmount());


describe('ResetPassword UI', () => {
  describe('render features', () => {
    test('container should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
