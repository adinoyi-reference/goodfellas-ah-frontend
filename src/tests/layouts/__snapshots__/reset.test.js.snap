// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword UI render features container should render as expected 1`] = `
<_default>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <MemoryRouter
      initialEntries={
        Array [
          Object {
            "key": "testKey",
          },
        ]
      }
    >
      <Router
        history={
          Object {
            "action": "POP",
            "block": [Function],
            "canGo": [Function],
            "createHref": [Function],
            "entries": Array [
              Object {
                "hash": "",
                "key": "testKey",
                "pathname": "/",
                "search": "",
              },
            ],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "index": 0,
            "length": 1,
            "listen": [Function],
            "location": Object {
              "hash": "",
              "key": "testKey",
              "pathname": "/",
              "search": "",
            },
            "push": [Function],
            "replace": [Function],
          }
        }
      >
        <Auth>
          <div
            className="auth-flex"
          >
            <AuthImage>
              <div
                className="author-writing"
              >
                <p
                  className="auth-header"
                >
                  Authors Haven
                </p>
                <img
                  alt=""
                  src="https://res.cloudinary.com/drmmqcxkc/image/upload/v1541518106/author.jpg"
                />
                <p
                  className="auth-title"
                >
                  A community of like minded authors
                </p>
              </div>
            </AuthImage>
            <div
              className="auth-form"
            />
          </div>
        </Auth>
      </Router>
    </MemoryRouter>
  </Provider>
</_default>
`;
